assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -325,6 +331,33 @@ const Cart = ( {
 								'woo-gutenberg-products-block'
 							) }
 							value={ parseInt( cartTotals.total_price, 10 ) }
+							description={
+								DISPLAY_CART_PRICES_INCLUDING_TAX && (
+									<p className="wc-block-cart__totals-footer-tax">
+										{ __experimentalCreateInterpolateElement(
+											__(
+												'Including <taxAmount/> in taxes',
+												'woo-gutenberg-products-block'
+											),
+											{
+												taxAmount: (
```

Comment:
It's preferable to use `TaxAmount` here because this is a component (even though this is an interpolated string).